docs/openj9_signals.md

```diff
@@ -71,6 +71,7 @@ Note that certain signals on VM threads cause OpenJ9 to shutdown. An application
 
 - The use of `SIGRTMIN` is configurable with the `-Xdump:suspendwith=<num>` option.
 - The handling of `SIGABRT` is configurable with the `-XX[+|-]HandleSIGABRT` option.
+- The handling of `SIGUSR2` is configurable with the [`-XX[+|-]HandleSIGUSR2`](xxhandlesigusr2.md) option.
```

Comment:
The colon is missing after `-XX` for both `ABRT` and `USR2`, here and several other places.

Reply:
When fixing, links could also be added for HandleSIGABRT and -Xdump:suspendwith

Reply:
I'll update https://github.com/eclipse-openj9/openj9-docs/pull/1084.